src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala

```diff
@@ -72,16 +73,20 @@ class PushPlatformValidationScenario(minMessagesInMailbox: Int,
       .exec(websocketConnect().onConnected(
         exec(enablePush)
           .during(duration) {
-            randomSwitch(
-              2.0 -> inboxHomeLoading.inboxHomeLoading,
-              8.0 -> selectArbitrary.selectArbitrary,
-              5.0 -> sendMessage(recipientFeeder),
-              30.0 -> openArbitrary.openArbitrary,
-              10.0 -> flagUpdate,
-              15.0 -> getNewState,
-              30.0 -> exec())
-              .asInstanceOf[ChainBuilder]
-              .pause(minWaitDelay, maxWaitDelay)
+            exec(
+              randomSwitch(
+                2.0 -> inboxHomeLoading.inboxHomeLoading,
+                8.0 -> selectArbitrary.selectArbitrary,
+                5.0 -> sendMessage(recipientFeeder),
+                30.0 -> openArbitrary.openArbitrary,
+                10.0 -> flagUpdate,
+                15.0 -> getNewState,
+                30.0 -> exec())
+                .asInstanceOf[ChainBuilder]
+                .pause(minWaitDelay, maxWaitDelay),
+
+              exec(echoPingWs)
+                .pause(pingInterval))
```

Comment:
From what I understood, if I put multiple chains in an exec, they execute in parallel? But I'm not exactly sure... Guess we will see :)

Reply:
No idea...

Reply:
Conclusion: this was a terrible idea! Somehow it makes it opening much more websockets than the number of users running... Really odd. Trying to find other ways...